substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -1019,6 +1019,7 @@ def _native_image_launcher_extra_jvm_args():
 ] + svm_experimental_options([
     '-H:IncludeResources=com/oracle/svm/driver/launcher/.*',
     '-H:-ParseRuntimeOptions',
+    f'-R:MaxHeapSize={256 * 1024 * 1024}',
```

Comment:
Shouldn't this go into the bash launcher as well? My understanding is this would only affect the AOT compiled `native-image` binary at runtime, right?

Reply:
That's correct, we don't have any good datapoints on how large this should be on the JVM. Does Mandrel set a max heap already?

Reply:
The native driver uses the Serial GC which uses 80% of total memory as max heap by default, the HotSpot GC use a much smaller share. So unless we have a problem with bash launchers and a good understanding how much memory the driver needs on HotSpot, let's set a max heap for the native driver for now?

Reply:
Sure.